docs/integrate/etl/influxdb.md

```diff
@@ -0,0 +1,191 @@
+
+    # From line protocol file to InfluxDB API.
+    influxio copy \
+        "file://tests/testdata/basic.lp" \
+        "http://example:token@localhost:8086/testdrive/demo"
+
+    # From line protocol file to InfluxDB API.
+    influxio copy \
+        "https://github.com/influxdata/influxdb2-sample-data/raw/master/air-sensor-data/air-sensor-data.lp" \
+        "http://example:token@localhost:8086/testdrive/demo"
+
+    # From line protocol file to any database supported by SQLAlchemy.
+    influxio copy \
+        "file://export.lp" \
+        "sqlite://export.sqlite?table=export"
+:::
+
+(cmd-export-from-api)=
+### Export from api
+
+:::{code} console
+    # From API to database file.
+    influxio copy \
+        "http://example:token@localhost:8086/testdrive/demo" \
+        "sqlite:///export.sqlite?table=demo"
+
+    # From API to database server.
+    influxio copy \
+        "http://example:token@localhost:8086/testdrive/demo" \
+        "crate://crate@localhost:4200/testdrive?table=demo"
+
+    # From API to line protocol file.
+    influxio copy \
+        "http://example:token@localhost:8086/testdrive/demo" \
+        "file://export.lp"
+
+    # From API to line protocol on stdout.
+    influxio copy \
+        "http://example:token@localhost:8086/testdrive/demo" \
+        "file://-?format=lp"
+:::
+
+(cmd-export-from-data-dir)=
+### Export from data directory
+
+:::{code} console
+    # From InfluxDB data directory to line protocol file.
+    influxio copy \
+        "file:///path/to/influxdb/engine?bucket-id=372d1908eab801a6&measurement=demo" \
+        "file://export.lp"
+
+    # From InfluxDB data directory to line protocol file, compressed with gzip.
+    influxio copy \
+        "file:///path/to/influxdb/engine?bucket-id=372d1908eab801a6&measurement=demo" \
+        "file://export.lp.gz"
+
+    # From InfluxDB data directory to line protocol on stdout.
+    influxio copy \
+        "file:///path/to/influxdb/engine?bucket-id=372d1908eab801a6&measurement=demo" \
+        ""file://-?format=lp"
+:::
+
+(cmd-oci)=
+### OCI 
+
+OCI images are available on the GitHub Container Registry (GHCR).
+In order to run them on Podman or Docker, invoke:
+
+:::{code} console
+docker run --rm --network=host ghcr.io/daq-tools/influxio \
+    influxio copy \
+    "http://example:token@localhost:8086/testdrive/demo" \
+    "stdout://export.lp"
+:::
+
+If you want to work with files on your filesystem, you will need to either
+mount the working directory into the container using the --volume option,
+or use the --interactive option to consume STDIN, like:
+
+:::{code} console
+docker run --rm --volume=$(pwd):/data ghcr.io/daq-tools/influxio \
+    influxio copy "file:///data/export.lp" "sqlite:///data/export.sqlite?table=export"
+
+cat export.lp | \
+docker run --rm --interactive --network=host ghcr.io/daq-tools/influxio \
+    influxio copy "stdin://?format=lp" "crate://crate@localhost:4200/testdrive/export"
+:::
```

Comment:
It does not make sense to copy that README-level documentation into another repository. It will only get stale too quickly. Please just link to it.

```suggestion
```